sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/AppendBlobAsyncClient.java

```diff
@@ -226,4 +230,22 @@
             .map(rb -> new SimpleResponse<>(rb, new AppendBlobItem(rb.getDeserializedHeaders(),
                 rb.getHeaders().value("x-ms-request-server-encrypted"))));
     }
+
+    /**
+     * Get the append blob name.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.AppendBlobAsyncClient.getName}
+     *
+     * @return The name of the append blob.
+     */
+    public String getNameAsync() {
```

Comment:
Given this class is a subclass of `BlobAsyncClient` we don't need to have a `getName` method here, it will be inherited.

Reply:
Done